### ISSUE TITLE
OpenApiResponseBody attribute with body type CreatedResult throws

### ISSUE THREAD

**Issue by gerab** [aliencube/AzureFunctions.Extensions#62]
`[OpenApiResponseBody(HttpStatusCode.Created, "application/json", typeof(CreatedResult), Summary = "CreatedResult object response.")]`

Log:
```
Executing 'RenderSwaggerDocument' (Reason='This function was programmatically called via the host APIs.', Id=xxxx)
Process is terminating due to StackOverflowException.
```

TargetFramework: netcoreapp2.2
AzureFunctionsVersion: v2
OpenApi: v1.5.4

**Comment by hf-kklein** [comment]
probably related to #55

**Comment by justinyoo** [comment]
It's been fixed from `1.6.1`. Please have a try it. I'm going to close this this issue. If you have the same problem, please create a new issue and link to this one.
